src/module/classes/cleric.mjs

```
@@ -1,23 +1,43 @@
- * Adds functionality to Healing Overflow
- *   @param {object[]} item - The Actor that rolled initiative.
- *   @param {string} roll - The resulting roll.
+ * Adds functionality to Infused Healer.
+ *   @param {object} actor - The actor using the item.
+ *   @param {object} activity - The activity performed.
+ *   @param {object} usageConfig - The usage configuration.
  */
-export function healOver(actor,combatants){    
-    console.log("Nothing ")
+export function infuseHeal(activity, usageConfig) {
+    let actor = activity.actor;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'actor' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)